spl-ids/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "solana-spl-ids"
+description = "Solana spl ids"
```

Comment:
nit: how about "Limited types and ids from the Solana Program Library"?

Reply:
done